vaadin-icons-flow-parent/vaadin-icons-flow/src/main/java/com/vaadin/flow/component/icon/Icon.java

```diff
@@ -119,9 +116,9 @@ public void setSize(String size) {
      */
     public void setColor(String color) {
         if (color == null) {
-            getStyle().remove(ElementConstants.STYLE_COLOR);
+            getStyle().remove(STYLE_FILL);
```

Comment:
Changed this API to use `fill` directly since `<vaadin-icon>` doesn't use `currentColor` as the default value for `fill` the way [iron-icon does](https://github.com/PolymerElements/iron-icon/blob/c0d4771e62704ac3c9d402f3d8e6ade53ac4a6fd/iron-icon.js#L98)

Reply:
Is it intentional that `<vaadin-icon>` does not use `currentColor` as the default fill color for the icons?
It sounds like a convenient feature that icons are affected in the same way as text by the CSS `color` property inherited from a parent. If it was not intentional, I suggest to update `<vaadin-icon>`.

Reply:
I'm not sure if it's intentional or not. Asked about it [here](https://vaadin.slack.com/archives/C020AE8T02Z/p1623413313140400?thread_ts=1623413300.140200&cid=C020AE8T02Z).